examples/node/lib/rest/api/V2010.ts

```diff
@@ -29,10 +31,16 @@ export default class V2010 extends Version {
     }
 
     protected _accounts?: AccountListInstance;
+    protected _credential?: CredentialListInstance;
```

Comment:
Should this be credentials?

Reply:
nerp: https://github.com/twilio/twilio-oai-generator/blob/578de68244482fa90fdaa620ec7fd41543b509e2/examples/twilio_api_v2010.yaml#L124